ct/zerotier-one.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
+# Copyright (c) 2021-2025 community-scripts ORG
+# Author: tremor021
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+
+# App Default Values
+APP="Zerotier-One"
+var_tags="networking"
+var_cpu="1"
+var_ram="512"
+var_disk="4"
+var_os="debian"
+var_version="12"
+var_unprivileged="1"
+
+# App Output & Base Settings
+header_info "$APP"
+base_settings
+
+# Core
+variables
+color
+catch_errors
+
+function update_script() {
+    header_info
+    check_container_storage
+    check_container_resources
+
+    if [[ ! -f /usr/sbin/zerotier-one ]]; then
+        msg_error "No ${APP} Installation Found!"
+        exit
+    fi
+    msg_info "Stopping Service"
+    systemctl stop zerotier-one &>/dev/null
```

Comment:
```suggestion
    msg_info "Stopping Service"
    systemctl stop zerotier-one &>/dev/null
    msg_ok "Stopping Service"
```
Each msg_info needs to be followed with a msg_ok, or you spawn a second spinner.